### DISCUSSION TITLE
Date & Time Stamp not useful for American calendars

### DISCUSSION THREAD

**Discussion by avenuesrini** [advanced-chat/advanced-chat-components#425] [Ideas]
@antoine92190  wonderful plug in congratulations. Let us know if we can support you in Patreon. 

I had a question on date formatting.. right now its international - DD/MM/YY  - is there any way to make it compatible for America too we can do it at Feb 6, '23 , 2:02 PM -- something like that. so we don't have to get confused in international and domestic date formats.

**Comment by antoine92190** [comment]
Let discuss this over https://github.com/antoine92190/vue-advanced-chat/discussions/424
